tests/FileLockApp/FileLockApp.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
```

This crashes the build on Mac and Linux, as net472 isn't present there.

```suggestion
   <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
    <TargetFrameworks Condition="$([MSBuild]::IsOsPlatform('Windows'))">$(TargetFrameworks)net472</TargetFrameworks>
        
```

---

Might be easier if I just remove this commit if it is causing problems - which it seems to be with the build errors. It is not a big problem to add the other framework to the csproj manually to test on other platforms if needed.

---

Removed the file lock app change.